pkg/cmd/kind/version/version_test.go

```diff
@@ -60,3 +60,59 @@ func TestTruncate(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion_Success(t *testing.T) {
+	tests := []struct {
+		name           string
+		gitCommit      string
+		gitCommitCount string
+		want           string
+	}{
+		{
+			name:           "With git commit count and with commit hash",
+			gitCommit:      "mocked-hash",
+			gitCommitCount: "mocked-count",
+			want:           versionCore + "-" + versionPreRelease + "." + "mocked-count" + "+" + "mocked-hash",
```

Comment:
IMO we should fully control the inputs and have a specific, static expected output (IE a hardcoded string, not a computed result), otherwise we're just writing the logic twice. That can be a followup.